src/meetup_manager/meetup_manager/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for meetup_manager project.
+
+Generated by 'django-admin startproject' using Django 3.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+import os
+import uuid
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve(strict=True).parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY', str(uuid.uuid4()))
```

Comment:
this is not a good idea, if the secret key is not provided in the .env file, the secret_key will be calculated in the runtime, 

consider this situation
when the user starts the server a new **secret_key** will be generated
the user logins to the admin dashboard
user decides to change the code and makes some changes, the server will be reloaded which means the **secret_key** will be calculated again
the current user logged in session will be invalidated, and the user needs to login again 

reference https://stackoverflow.com/questions/15170637/effects-of-changing-djangos-secret-key

this would be better 

```SECRET_KEY = os.getenv('DJANGO_SECRET_KEY', "secret_key_for_development")```

P.S
I have pulled this code and tested it :)

Reply:
Yes, changing the secret_key frequently is not recommended and it's definitely not something that should be done in real-time.

In this case as well, we wanted to explicitly set the expectation that the any user / dev needs to set their own secret key for development as well, instead of setting a static key for all users, but I didn't want it to fail in case it wasn't provided.

I missed to add this to the README while making this change.

Reply:
cool 👍
LGTM

Reply:
cool 👍
LGTM

Reply:
added the key to `.env.development` file